packages/app/features/dashboard/search/recently-added/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { FlatList } from "react-native";
+import { Text } from "app/design-system";
+import { useEffect, useState } from "react";
+import { Entry, useRecentlyAddedQuery } from "app/api/graphql";
+import { BeatListEntry } from "app/features/dashboard/search/recently-added/list-entry";
+
+export default function RecentlyAddedList() {
+  const [nextPage, setNextPage] = useState(0);
+  const [data, setData] = useState<Entry[]>([]);
+  const { data: queryData, refetch } = useRecentlyAddedQuery({
+    variables: { page: 0 },
+  });
+
+  useEffect(() => {
+    if (queryData && queryData.recentlyAdded) {
+      const recentlyAdded: Entry[] = queryData.recentlyAdded.filter(
+        (entry): entry is Entry => entry !== null
```

Comment:
add link to stack

Reply:
https://stackoverflow.com/questions/43010737/way-to-tell-typescript-compiler-array-prototype-filter-removes-certain-types-fro